src/targetValidationUtils.js

```diff
@@ -97,10 +104,19 @@ function isOverValidBoundaryErrorEventTarget(clientX, clientY, paper, graph) {
   return true;
 }
 
+function isOverValidBoundaryEscalationEventTarget(clientX, clientY, paper, graph) {
+  return graph.findModelsFromPoint(getLocalMousePosition(clientX, clientY, paper)).
+    find(({ component }) => component && validBoundaryEscalationEventTarget.includes(component.node.definition.$type));
```

Comment:
This line here is the same as the one you have in `boundaryEscalationEvent.vue`:
```js
model.component && validBoundaryEscalationEventTargets.includes(model.component.node.definition.$type);
```

Few things:
 * You call it `validBoundaryEscalationEventTarget` (singular) here, but `validBoundaryEscalationEventTargets` (plural) on other places.
 * `validBoundaryEscalationEventTarget` is an array wth a single item; I don't think it needs to be an array.

Since you use the same check in a couple places, I think it might be better to extract that into a util function instead of having the single item array be the extraction. The function could be something like:
```js
function isValidBoundaryEscalationEvent(model) {
  model.component && model.component.node.definition.$type === 'bpmn:CallActivity'
}
```